### ISSUE TITLE
Selecting a visualization for a column should not require clicking the check box

### ISSUE THREAD

**Issue by alexsb** [lineupjs/lineupjs#457]
When I choose a different visualization for a column, I see a preview of that immediately in lineup. 
When I then click on the lineup view to make the menu disappear, the column reverts to the old visualization. 
To confirm a visualization, I have to click a small check mark at the bottom: 
![image](https://user-images.githubusercontent.com/2039375/126648303-7309b341-8657-46f3-bb0b-9dcdad75ee13.png)
 

**Expected behavior**

When I click away, the state of the preview is retain. That means, "apply" is the default action, not "cancel". 
I don't think we need an undo button here. The undo button also doesn't do an expected "undo", but resets to the initial state. I can just as well cancel. 
This should use proper buttons with suggested action highlight.

**Comment by thinkh** [comment]
We unified all dialogs and added the live preview feature with PR #265. Hence, you need to confirm the changes with the tickmark at the bottom. If we want to change this behavior (again), we should asses it for all dialogs to avoid specific dialogs behaviors again.

**Comment by alexsb** [comment]
I think this is all fine, but I do think that the default should be accept, not reject the changes. Or why should it not be like that? Can you give me an example where this is odd?

**Comment by thinkh** [comment]
> I think this is all fine, but I do think that the default should be accept, not reject the changes. Or why should it not be like that? Can you give me an example where this is odd?

Yes, makes sense. Should be just changing the default value here: https://github.com/lineupjs/lineupjs/blob/5af9d9b9c3a9f6b68b105c545cf0509267d8dbac/src/ui/dialogs/ADialog.ts#L238

> And how is reset useful?
Take a histogram filter of a numeric column. Instead of fiddling around to drag the left and right handlers to it's position, you can just click on the reset button. Same goes for data mapping editor.

**Comment by alexsb** [comment]
Re 1 - yes. 

Re 2 - ok. Not sure how much better this is than just cancelling the whole operation and re-opening the dialog, especially given that it's not an "undo" as the button suggests but a "reset". Maybe in an ideal world, we would be able to do proper undo?

And if we want to keep it, we should distinguish between simple dialogs that don't need that (like view specification) and more complex ones where it might make sense.

**Comment by thinkh** [comment]
Maybe the icon is missleading and we need a "reset" icon? Because the tooltip states clearly that it resets all values.

![grafik](https://user-images.githubusercontent.com/5851088/126659929-7a83ef6d-5c2c-44f3-93c6-e1a0347549e8.png)
